src/runtime/providers/vercel.ts

```diff
@@ -5,7 +5,7 @@ export interface Options {
   debug: boolean
 }
 
-const vitalsUrl = 'https://vitals.vercel-analytics.com/v1/vitals'
+const vitalsUrl = 'https://vitals.vercel-analytics.com/v1/vitals?d=1'
```

Comment:
I'm not sure there's any need to modify this URL - is there?
```suggestion
const vitalsUrl = 'https://vitals.vercel-analytics.com/v1/vitals'
```

Reply:
Yes, But this is just a dummy GET parameter at start to be compatible with `&` symbol on `encodeParams(...)` in `src/runtime/utils.ts`.

Reply:
Ah, understood. `encodeParams` is not used in this runtime provider, however, so I think we can safely omit this change.

Reply:
I see, you are right.